src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareAsStrings) {
+    let count;
+
+    do {
+      count = 0;
+
+      for (let i = 1; i < this.length; i++) {
+        const prev = this[i - 1];
+        const current = this[i];
+
+        if (compareFunction(prev, current) > 0) {
+          this[i] = prev;
+          this[i - 1] = current;
+          count++;
+        }
+      }
+    } while (count > 0);
+
+    return this;
+  };
+
+  const compareAsStrings = (a, b) => {
+    const stringA = String(a);
+    const stringB = String(b);
+
+    if (stringA > stringB) {
+      return 1;
+    } else if (stringA === stringB) {
+      return 0;
+    } else {
+      return -1;
+    }
```

Comment:
There is no need to use else...if. Your code will not go further after return anyway
```suggestion
    if (stringA > stringB) {
      return 1;
    }
    
    if (stringA < stringB) {
      return -1;
    } 
    
    return 0;
```